tests/modules/nf-core/ilastik/multicut/main.nf

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ILASTIK_MULTICUT } from '../../../../../modules/nf-core/ilastik/multicut/main.nf'
+
+workflow test_ilastik_multicut {
+
+    input = [
+        [ id:'probabilities' ], // meta map
+        file("/Users/florian_wuennemann/1_Projects/nf_core/test_data/wga.h5")
+    ]
+    ilp = [file("/Users/florian_wuennemann/1_Projects/nf_core/test_data/multicut_140rc8_plant.ilp")]
+    probs = [file("/Users/florian_wuennemann/1_Projects/nf_core/test_data/wga-data_Probabilities.h5")]
```

Comment:
Can you use online available data here? Preferably data that is in the `nf-core/test-datasets` repository. Otherwise nobody else can run these tests :)

Reply:
I just submitted a PR to add these data to test-datasets. Sorry, I made these PRs before submitting the test-data PR!

https://github.com/nf-core/test-datasets/pull/771

Will fix the file paths as soon as this is merged!

Reply:
Test-data is now available and paths have been fixed!